server.js

```diff
@@ -1,6 +1,7 @@
 const express = require('express')
 const port = Number(process.env.PORT) || 5000
-const repos = require('all-the-package-repos')
+const obj2map = obj => Object.keys(obj).reduce((m, k) => m.set(k, obj[k]), new Map())
+const repos = obj2map(require('all-the-package-repos'))
```

Comment:
Just a minor note: I think we can free the object from memory using `delete require.cache[require.resolve("all-the-package-repos")]`

Not sure if it matters at all, but it will get rid of the duplicated content from the memory. :grin:

Reply:
Sounds good!